.github/workflows/test.yml

```diff
@@ -9,20 +9,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby_version: ["2.3.8", "2.4.5", "2.5.3", "2.6.3", "2.7.2"]
+        ruby_version: ["2.3", "2.4", "2.5", "2.6", "2.7", "3.0", "3.1"]
         experimental: [false]
         include:
-          - ruby_version: "3.0"
+          - ruby_version: "3.2"
             experimental: true
           - ruby_version: "ruby-head"
             experimental: true
-          - ruby_version: "jruby-head"
-            experimental: true
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby_version }}
-      - run: "bundle install"
+          bundler-cache: true
```

Comment:
Use the `ruby/setup-ruby` action to run `bundle install` and cache the gems between builds for better performance.